RoCur documentation update (#80)
rladies/rladiesguide: content/rocur/admin guide/index.en.md

   - Alt Text: When uploading an image into Tweetdeck, a gray button will appear at the bottom of the image labelled "Add description." In the text field, we currently write:   
   > Curator **[NAME]**, **[Role/Title]** at **[Organization]**. They say: "**[copy text about what they do in R]**"  
* They will need to visit [tweetdeck.twitter.com](https://tweetdeck.twitter.com/) while logged into their twitter account in order to tweet from the account.  
  

> #### Right before they start their curation:  
> * Change the profile picture for the account to the photo provided by the curator.  
> * In the description, write their Twitter handle after "This week's curator:"  

Reviewer: ```suggestion
#### Immediately prior to curation:  

* Change the profile picture for the account to the photo provided by the curator.  
* In the description, write their Twitter handle after "This week's curator:"  
```

>If you have any questions or encounter any problem, you can send us an email at WeAre@rladies.org. Also, you can join our #rocur_twitter channel on the community slack where you can reach out to all RoCur Managers and get connected with past curators.  
>  
>Thanks!  
  
### After they curate  
   

> #### During the Saturday after their curation ends:  

Reviewer: ```suggestion
#### The day after their curation ends:  
```

As not all curations end on Saturday??
